### ISSUE TITLE
Translation of the navigation bar

### ISSUE THREAD

**Issue by ivanhercaz** [SemanticMediaWiki/semantic-mediawiki.org#82]
Hi everyone,

Is there any way to translate the items of the navigation bar configured in [MediaWiki:Sidebar](https://www.semantic-mediawiki.org/wiki/MediaWiki:Sidebar)? It would be really nice to have it in another languages to ease the non-English speakers to learn about Semantic MediaWiki.

Regards,
Iván

**Comment by kghbln** [comment]
Yes, however I have not looked into the matter on how to add respective system messages to the translation system without having to grant translators admin rights. That's basically the big issue here. Perhaps transclusion of templates in the system message is a way out.
